### PR TITLE
Enhance readme documentation

### PR DESCRIPTION
**Pull request by lalmeras**
More information about direct resource delivery by Servlet 3 container and  on explicit version.
Some insight on the expected format in Webjars*ResourceReference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,6 +113,8 @@ public WebjarsComponent extends Panel {
 }
 ```
 
+**Note**: you must specify in your path either an explicit version (``1.8.3``) or the ``<recentVersionPlaceHolder>`` configured in ``IWebjarsSettings`` (``current`` for a default config).
```
Comment (martin-g) [inline README.md]: Please change `(``1.8.3``)` to `(e.g. ``1.8.3``)`

**Comment by lalmeras** [comment]
Done. Do you want I merge the two commits ?

**Comment by martin-g** [comment]
Thank you, @lalmeras !
GitHub UI allows to squash before the merge.

**Comment by martin-g** [comment]
Ported it to `wicket-8.x` !

**Pull request merged**